_episodes/03-configuration.md

```diff
@@ -3,91 +3,178 @@ title: "Configuration"
 
-## Output settings
+~~~YAML
+drs:
+  CMIP5: default
+~~~
+
+> ## Available drs
+>
+> For more information about directories, you can visit ESMValTool [documentation](https://esmvaltool.readthedocs.io/projects/esmvalcore/en/latest/esmvalcore/config.html#developer-configuration-file).
+{: .callout}
+
+## Number of parallel tasks
 
-.. code-block:: yaml
+This option enables you to perform parallel processing. You can choose the number of tasks in parallel as 1/2/3/4/... or you can set it to ``null`` that tells ESMValTool to use the number of available CPUs:
 
-  # Diagnostics create plots? [true]/false
-  write_plots: true
-  # Diagnositcs write NetCDF files? [true]/false
-  write_netcdf: true
+~~~YAML
 
-The ``write_plots`` setting is used to inform ESMValTool about your preference
-for saving figures. Similarly, the ``write_netcdf`` setting is a boolean which
-turns on or off the writing of netCDF files.
+max_parallel_tasks: null
+~~~
 
-The ```rootpath`` specifies the directories where ESMValTool will look for input
-data. Similarly, ``output_dir`` specifies where ESMValTool will store its
-output, i.e. figures, data, logs, etc. Make sure to set appropriate paths.
+> ## Set the number of tasks
+>
+> If you run out of memory, try setting ``max_parallel_tasks`` to 1. Then, check the amount of memory you need for that by inspecting the file ``run/resource_usage.txt`` in the output directory. Using the number there you can increase the number of parallel tasks again to a reasonable number for the amount of memory available in your system.
+{: .callout}
 
-.. code-block:: yaml
+## Destination directory
 
+The destination directory is the rootpath where ESMValTool will store its output, i.e. figures, data, logs, etc. With every run, ESMValTool automatically generates a new output folder determined by the recipe name, and date and time using the format: YYYYMMDD_HHMMSS. This folder contains four further subfolders: ``plots``, ``preproc``, ``run``, ``work``.
 
+Let's name our destination directory as ``esmvaltool_output`` in the working directory:
 
+~~~YAML
+output_dir: ./esmvaltool_output
+~~~
 
-You can tailor it for your system using the explanation below.
+> ## Content of subfolders
+>
+> * ``plots``: the location for all plots, split by individual diagnostics and fields.
+> * ``preproc``: this folder contains all the preprocessed data and metadata.yml interface files. Note that by default this directory will be deleted after each run because most users will only need the results from the diagnostic scripts.
+> * ``run``: this folder includes all log files, a copy of the recipe, a summary of the resource usage, and the settings.yml interface files, resource_usage.txt and temporary files created by the diagnostic scripts.
+> * ``work``: a place for any diagnostic script results that are not plots, e.g. files in NetCDF format (depends on the diagnostic script).
```

Comment:
I would rephrase to be consistent with the entries before "``work``: this folder is a place for any..."